src/query/runner/ipa.rs

```diff
@@ -115,6 +115,7 @@ mod tests {
 
         let world = TestWorld::default();
         let contexts = world.contexts();
+        #[allow(clippy::large_futures)]
```

Comment:
The future is 23024 bytes, vs. a limit of 16 kB. A limit of 16 kB for futures on the stack seems unnecessarily conservative for us, but since this is a test anyways, I just waived it.

Reply:
I love this new lint, I don't have to guess the future size on the stack anymore!